src/Form/ViewModeMappingSettingsForm.php

```diff
@@ -55,7 +56,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $config = $this->config('format_strawberryfield.viewmodemapping_settings');
     $form['info'] = [
       '#markup' => $this->t(
-        'This Form allows you to map ADO (Archipelago Digital Object) "types" to existing Drupal View Mode Configurations.'
+        'This Form allows you to map ADO (Archipelago Digital Object) "types" to Drupal View Modes.'
```

Comment:
OK, I can see how this is shorter. Though the "existing" part is not wrong!

Reply:
Not wrong, but not necessary either. Is it a view mode if it doesn't exist? :grinning: